velox/functions/prestosql/Map.cpp

```diff
@@ -358,4 +358,12 @@ VELOX_DECLARE_VECTOR_FUNCTION(
     udf_map_allow_duplicates,
     MapFunction</*AllowDuplicateKeys=*/true>::signatures(),
     std::make_unique<MapFunction</*AllowDuplicateKeys=*/true>>());
+
+void registerMapFunction(const std::string& name, bool allowDuplicateKeys) {
```

Comment:
What's the use case for `allowDuplicateKeys=true`?

Reply:
Used in Spark: velox/functions/sparksql/Register.cpp